src/engine/util/util_draw.cpp

```diff
@@ -55,16 +55,16 @@ namespace Util
 
      //--------------------------------------------------------
     void
-    DrawGrid(SDL_Renderer *renderer, Uint32 spacing_x, Uint32 spacing_y, float start_x, float start_y, Uint32 n_rows, Uint32 n_cols,
+    DrawGrid(SDL_Renderer *renderer, float spacing_x, float spacing_y, float start_x, float start_y, Uint32 n_rows, Uint32 n_cols,
```

Comment:
why change from uint to float?

Reply:
Need exact position for the spacing since we no longer deal with nice whole integers when the size of the texture and the # of rows and cols don't split everything up evenly.